org-static-blog.el

```diff
@@ -312,7 +318,37 @@ e.g. `(('foo' 'file1.org' 'file2.org') ('bar' 'file2.org'))`"
               (push post-filename (cdr (assoc-string tag tag-tree t)))
             (push (cons tag (list post-filename)) tag-tree)))))
     tag-tree))
-
+(defun org-static-blog-get-preview (post-filename)
+  (with-temp-buffer
+    (insert-file-contents (org-static-blog-matching-publish-filename post-filename))
+    (let ((title-start)
+          (paragraph-end)
+          (post-start)
```

Comment:
"post" is referring to the postamble, right? If so, how about `postamble-start` and `postamble-end`, to avoid confusing "blog posts" with postambles.